### PR TITLE
animation loop option to pause on max frame, not reset to beginning

### PR DESCRIPTION
**Pull request by abenrob**
Per https://github.com/CartoDB/torque/issues/140#issuecomment-76687364
Modifying Animator._tick so loop=false pauses animation on final frame, not stops and resets to beginning.
57 tests still passing.

### CODE REVIEW

File: lib/torque/animator.js [hunk 1]
```diff
@@ -118,9 +118,15 @@ var cancelAnimationFrame = global.cancelAnimationFrame
       this._t0 = t1;
       this._time += delta;
       if(this.step() >= this.options.steps) {
-        this._time = 0;
         if(!this.options.loop){
-          this.stop();
+          // set time to max time
+          this.time(this.options.animationDuration);
+          // go to final frame (callback to nothing)
+          requestAnimationFrame(function(){});
```
Comment (javisantana) [inline lib/torque/animator.js]: is this needed?

Comment (abenrob) [inline lib/torque/animator.js]: Without it, the animation might stop on a frame that is not the final. The animation does not call every single step - for this reason, I couldn't intercept the last frame in the client library. In my case, I'm using the SQL api to call the table and display counts, so if we stop on, say 506 of 512, the counts for that timestamp wont be the final.

Comment (javisantana) [inline lib/torque/animator.js]: but calling time should render the last frame (https://github.com/abenrob/torque/blob/loop-pause-max/lib/torque/animator.js#L66)

Comment (abenrob) [inline lib/torque/animator.js]: Brilliant! I totally missed that!

**Comment by javisantana** [comment]
good, @fdansv I think it would be nice to have that param documented

**Comment by fdansv** [comment]
yep! also to tag since we've merged dists :stuck_out_tongue:

**Pull request merged**